public/docs/ts/latest/guide/architecture.jade

```diff
@@ -392,151 +438,160 @@ figure
   The `HeroService` depends on the `Logger` service and another `BackendService` that handles the server communication grunt work.
-+makeExample('architecture/ts/app/hero.service.ts', 'class', 'app/hero.service.ts (class only)')(format=".")
+
++makeExcerpt('app/hero.service.ts', 'class')
+
 :marked
-  Services are everywhere. 
+  Services are everywhere.
 
-  Our components are big consumers of services. They depend upon services to handle most chores. 
-  They don't fetch data from the server, they don't validate user input, they don't log directly to the console. 
+  We prefer our component classes lean. Our components don't fetch data from the server,
+  they don't validate user input, and they don't log directly to the console.
   They delegate such tasks to services.
-  
+
   A component's job is to enable the user experience and nothing more. It mediates between the view (rendered by the template)
-  and the application logic (which often includes some notion of a "model").  A good component presents
-  properties and methods for data binding. It delegates everything non-trivial to services.
+  and the application logic (which often includes some notion of a _model_).
+  A good component presents properties and methods for data binding.
+  It delegates everything nontrivial to services.
 
-  Angular doesn't *enforce* these principles. 
+  Angular doesn't *enforce* these principles.
   It won't complain if we write a "kitchen sink" component with 3000 lines.
-  
+
   Angular does help us *follow* these principles by making it easy to factor our
   application logic into services and make those services available to components through *dependency injection*.
 
 .l-main-section
-<a id="dependency-injection"></a>
 :marked
-  ## Dependency Injection
+  ## Dependency injection
 figure
   img(src="/resources/images/devguide/architecture/dependency-injection.png" alt="Service" style="float:left; width:200px; margin-left:-40px;margin-right:10px" )
 :marked
-  "Dependency Injection" is a way to supply a new instance of a class 
+  Dependency injection is a way to supply a new instance of a class
   with the fully-formed dependencies it requires. Most dependencies are services.
   Angular uses dependency injection to provide new components with the services they need.
 <br clear="all">
 :marked
-  In TypeScript, Angular can tell which services a component needs by looking at the types of its constructor parameters.
-  For example, the constructor of our `HeroListComponent` needs the `HeroService`:
-+makeExample('architecture/ts/app/hero-list.component.ts', 'ctor', 'app/hero-list.component (constructor)')(format=".")
+  Angular can tell which services a component needs by looking at the types of its constructor parameters.
+  For example, the constructor of our `HeroListComponent` needs a `HeroService`:
+
++makeExcerpt('app/hero-list.component.ts (constructor)', 'ctor')
```

Comment:
It seems like this transcript should include the place where _heroService is declared, so you can see it's a HeroService.